video/out/w32_common.c

```diff
@@ -2195,8 +2246,29 @@ struct disp_names_data {
     HMONITOR assoc;
     int count;
     char **names;
+    bool friendly;
 };
 
+static void disp_names_append(HMONITOR mon, struct disp_names_data* data)
+{
+    MONITORINFOEXW mi = { .cbSize = sizeof mi };
+    if (GetMonitorInfoW(mon, (MONITORINFO*)&mi)) {
+        wchar_t* friendly_name = NULL;
+        if (data->friendly) {
+            friendly_name =
+                mp_w32_displayconfig_get_friendly_name_from_device(mi.szDevice);
```

Comment:
We allow overhanging lines up to 100 columns, just to allow such cases to be not multi line.